packages/amplify-cli/src/commands/push.ts

```diff
@@ -61,7 +20,12 @@ export const run = async (context: $TSContext): Promise<$TSAny> => {
   if (context.parameters.options?.force) {
     context.exeInfo.forcePush = true;
   }
-  await syncCurrentCloudBackend(context);
+
+  // force flag ignores project status check
+  if (!context.exeInfo.forcePush) {
+    await syncCurrentCloudBackend(context);
+  }
```

Comment:
Should we add an e2e test where we nuke deployment bucket and then attempt to force push ?

Reply:
will do